WASM.md

```diff
@@ -1,23 +1,64 @@
-Proof of Concept WebAssembly Support.
+WebAssembly Extension Support
 
-# WebAssembly VM
+# Dependencies
```

Comment:
Perhaps split this into "build dependencies" and "dependencies for regenerating tests modules"?

Reply:
done.